Documentation/cmdref/cilium-agent.md

```diff
@@ -309,7 +309,7 @@ cilium-agent [flags]
       --l2-announcements-lease-duration duration                  Duration of inactivity after which a new leader is selected (default 15s)
       --l2-announcements-renew-deadline duration                  Interval at which the leader renews a lease (default 5s)
       --l2-announcements-retry-period duration                    Timeout after a renew failure, before the next retry (default 2s)
-      --l2-pod-announcements-interface string                     Interface used for sending gratuitous arp messages
+      --l2-pod-announcements-interfaces string                    Regex matching interfaces used for sending gratuitous arp messages
```

Comment:
Why do we lose the docs for `--l2-pod-announcements-interface` if, as I understand, we still support it in addition to `--l2-pod-announcements-interfaces`?

Reply:
Ah got it, that's because you marked it as deprecated :+1: